### DISCUSSION TITLE
Remove "set as background" button in media viewer

### DISCUSSION THREAD

**Discussion by Donkey-Doug** [Docile-Alligator/Infinity-For-Reddit#864] [Ideas]
Who even sets stuff from reddit as background?

**Comment by Wladefant** [comment]
Everyone who is interested in this enhancement please upvote. Right now we are deciding what the threshold of upvotes for feature requests is.

**Comment by MysteriousOrb2000** [comment]
I agree with this. The niche who do use this can just download the image and set it as wallpaper from their device. This is just taking up space unnecessarily.
